src/detect_breakends.cpp

```diff
@@ -28,6 +29,11 @@ void initialize_argument_parser(seqan3::argument_parser & parser, cmd_arguments
     //                                               "3", "read_pairs",
     //                                               "4", "read_depth"};
     seqan3::arithmetic_range_validator method_validator{1, 4};
+    // seqan3::value_list_validator clustering_method_validator{"0", "simple_clustering",
+    //                                                          "1", "hierarchical_clustering",
+    //                                                          "2", "self-balancing_binary_tree",
+    //                                                          "3", "candidate_selection_based_on_voting"};
```

Comment:
I don't understand what this comment is telling me and why are the numbers here strings?